ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,137 @@
+const copyStyle = `
+.code-n-btn-wrapper {
+  positon: relative;
+}
+.copy-code-wrapper {
+  display: none;
+  position: absolute;
+  right: 0;
+  margin: 10px 15px 0 0;
+}
+.copy-code-wrapper button {
+   background: #fff;
+   border: none;
+   padding: 10px;
+   cursor: pointer;
+}
+.copy-code-wrapper button i {
+font-size: 1rem;
+}
+.code-n-btn-wrapper:hover .copy-code-wrapper{
+  display: block;
+}
+.u-mdic-copy-notify {
+  display: none;
+  padding: 4px 12px;
+  background: #fff;
+  color: #ccc;
+  font-size:0.9rem;
+  margin: 0 10px;
+  border-radius: 10px;
+}
```

Comment:
Very light gray on white is very hard to read. Please use colour `#777` for instance.

Reply:
Alright